types/mui-datatables/mui-datatables-tests.tsx

```diff
@@ -157,15 +175,15 @@ const MuiCustomTable: React.FC<Props> = props => {
         },
     };
 
-    return <MUIDataTable title={props.title} data={data} columns={columns} options={TableOptions} />;
+    return <MUIDataTable title={props.title} data={data} columns={columns} options={TableOptions}/>;
```

Comment:
```suggestion
    return <MUIDataTable title={props.title} data={data} columns={columns} options={TableOptions} />;
```

Reverted autoformatting